docs/deploy-to-vercel.md

```diff
@@ -1,27 +1,13 @@
 # Deploy to Vercel
 
-[Vercel](https://vercel.com) (formerly ZEIT) is a cloud platform for websites and serverless APIs, that you can use to deploy your Gridsome projects to your personal domain (or a free `.now.sh` suffixed URL).
+[Vercel](https://vercel.com) is a cloud platform that enables developers to host Jamstack websites and web services that deploy instantly, scale automatically, and requires no supervision, all with zero configuration. They provide a global edge network, SSL encryption, asset compression, cache invalidation, and more.
 
-This guide will show you how to get started in a few quick steps:
+### Step 1: Deploying your Gridsome Website to Vercel
 
-### Step 1: Installing Now CLI
+To deploy your Gridsome app with a [Vercel for Git Integration](https://vercel.com/docs/git-integrations), make sure it has been pushed to a Git repository.
 
-To install their command-line interface with [npm](https://www.npmjs.com/package/now), run the following command:
+Import the project into Vercel using the [Import Flow](https://vercel.com/import/git). During the import, you will find all relevant options preconfigured for you; however, you can choose to change any of these options, a list of which can be found [here](https://vercel.com/docs/build-step#build-&-development-settings).
 
-```shell
-npm i -g now
-```
+After your project has been imported, all subsequent pushes to branches will generate [Preview Deployments](https://vercel.com/docs/platform/deployments#preview), and all changes made to the [Production Branch](https://vercel.com/docs/git-integrations#production-branch) (commonly "main") will result in a [Production Deployment](https://vercel.com/docs/platform/deployments#production).
```

Comment:
Would it be possible to rewrite this in the active voice? This paragraph omits all subjects, so it's confusing for the reader who's performing these actions.

Reply:
I'm not sure I follow here? It shouldn't matter who is doing the pushing — it is merely stating the fact regardless of who is specifically to their their Vercel project.

Reply:
Right, I mean there's ambiguity between Vercel and the user.

Some of the verbs in this paragraph refer to things that Vercel does ("generate", "result"), and some refer to things that the user does ("imported", "pushes", "changes"). The current wording makes it ambiguous which actions Vercel performs and which action the user performs.

Reply:
Fixing!